connectors/kafka/src/schemas/UserRegistered.avsc

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
I'd actually prefer to just get rid of or modify this, because it seems weird to have something with such a direct purpose being used for a test.